src/visualization/types/Graph/view.tsx

```diff
@@ -57,6 +57,7 @@ import {isFlagEnabled} from '../../../shared/utils/featureFlag'
 // Selectors
 import {getByID} from 'src/resources/selectors'
 import {updateViewAndVariables} from 'src/views/actions/thunks'
+import {DashboardColor} from '../../../client'
```

Comment:
nit: 
```suggestion
import {DashboardColor} from 'src/client'
```

Reply:
Dang it, I need to change my IDE settings. I'll fix this.